### PR TITLE
20-10207 - Add Flipper

### PR DESCRIPTION
**Pull request by tlei123**
## Summary

- Add Flipper to Priority Processing (20-10207)
  - NOTE: Final-URL & breadcrumbs, as well as React-route, already done via department-of-veterans-affairs/va.gov-team-forms#1026.
- Team: Veteran Facing Forms
- Flipper end-date TBD

## Companion PR

[vets-api](https://github.com/department-of-veterans-affairs/vets-api/pull/15898)

## Related issue(s)

- department-of-veterans-affairs/va.gov-team-forms#1083

## Testing done

- Local browser-testing was successful

## What areas of the site does it impact?

This form ONLY

### CODE REVIEW

**Review by rhasselle-oddball — DISMISSED** [review]

**Comment by jamigibbs** [comment]
Hi @tlei123, once tests are passing, let me know and I'll be happy to review for Platform FE CoP if needed.

**Review by rhasselle-oddball — DISMISSED** [review]

File: src/applications/static-pages/simple-forms/20-10207/App.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    return <va-loading-indicator message="Loading..." />;
+  }
+
+  if (formEnabled) {
+    return (
+      <>
+        <p>You can submit this form online or by mail.</p>
+        <a
+          className="vads-c-action-link--blue"
+          href="/supporting-forms-for-claims/request-priority-processing-form-20-10207"
```
Comment (jamigibbs) [inline src/applications/static-pages/simple-forms/20-10207/App.js:19]: You should be able to use the `va-link` component here: https://design.va.gov/storybook/?path=/docs/components-va-link--default

Comment (jamigibbs) [reply src/applications/static-pages/simple-forms/20-10207/App.js:19]: Disregard! I see that we don't have the [action link](https://design.va.gov/storybook/?path=/docs/components-action-link--primary) variation of this as a web component yet.

**Review by jamigibbs — DISMISSED** [review]

File: src/applications/static-pages/simple-forms/20-10207/App.js (hunk 1 above)
Comment (tlei123) [reply src/applications/static-pages/simple-forms/20-10207/App.js:19]: Replaced with va-link.  Thanks for catching!

**Review by jamigibbs — APPROVED** [review]

**Review by rhasselle-oddball — APPROVED** [review]

**Pull request merged**